packages/design-system-scripts/cli.js

```diff
@@ -164,6 +170,11 @@ yargs
           type: 'array',
           default: ['**/node_modules/**', '**/dist/**'],
         })
+        .option('failAfterError', {
```

Comment:
Previously we would have our linting fail on error if the env is set to `TEST`. Now it is just a command line option.